src/proximity/styles.css

```
@@ -0,0 +1,500 @@
+[data-proximity] {
+	--px-border: rgb(var(--gray-light));
+	--px-muted: rgb(var(--gray));
+	--px-ink: rgb(var(--gray-dark));
+	--px-title: rgb(var(--black));
+	--px-fill: rgb(var(--surface));
+	--px-wash: rgb(var(--bg));
+	box-sizing: border-box;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)